high_card.rb

```diff
@@ -1,5 +1,60 @@
-# Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+require 'pry'
+Ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A" ]
+Suits = [ "hearts", "spades", "clubs", "diamonds" ]
```

Comment:
i think you could change these to be regular variables instead of constants.  All you would need to do is be sure you're passing them into the necessary methods. will comment additionally below!